webpack.common.js

```diff
@@ -44,7 +44,6 @@ module.exports = {
     },
     dsgvo_video_embed: {
       import: [
-        'dsgvo-video-embed/css/dsgvo-video-embed.css',
```

Comment:
@vellip @hom3mad3 the css file produced by webpack (`dsgvo_video_embed.css`) is still used in various places. Removing this here will cause a 500 (https://meinberlin-design-dev.liqd.net/projekte/participatory-budgeting-3-support-test/). We need to either add it back in or remove it from the templates and use the new video-wall there as well